Source/WebKit/WebProcess/WebCoreSupport/WebWorkerClient.cpp

```diff
@@ -139,32 +151,32 @@ WebWorkerClient::~WebWorkerClient() = default;
 
 UniqueRef<WorkerClient> WebWorkerClient::createNestedWorkerClient(SerialFunctionDispatcher& dispatcher)
 {
-    assertIsCurrent(this->dispatcher());
+    assertIsCurrent(dispatcher);
```

Comment:
Not clear to me what guarantees that `this->dispatcher()` and `dispatcher` are equivalent here.

Reply:
Oh oops, this was a mistake. Reverted.